### PR TITLE
T1389 Fixed erroneous script tags

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1389-test-files-in-tempate-directory-contain-invalid-script-tag#ticket-1389-3

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fa4d548dbbac746/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/fa4d548dbbac746/output.txt

Total script time: 0.21 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/898545072e3e9ce/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/898545072e3e9ce/output.txt

Total script time: 0.49 mins

Server is running on http://pr430.butyr.org/templates/test.html

File: templates/complete.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <title>Basic template</title>
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <script src="https://browserid.org/include.js" type="text/javascript" data-butter-exclude></script>
-    <script src="../src/butter.js" data-butter-exclude /></script>
-    <script type="text/javascript" src="complete.js" data-butter-exclude /></script>
+    <script src="../src/butter.js" data-butter-exclude ></script>
+    <script type="text/javascript" src="complete.js" data-butter-exclude ></script>
```
Comment (ScottDowne) [inline templates/complete.html]: Should not be a space between the exclude and the >

File: templates/external-media.html [hunk 2]
```diff
@@ -4,8 +4,8 @@
     <title>Basic External Media Template</title>
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <script src="https://browserid.org/include.js" type="text/javascript" data-butter-exclude></script>
-    <script src="../src/butter.js" data-butter-exclude /></script>
-    <script type="text/javascript" src="external-media.js" data-butter-exclude /></script>
+    <script src="../src/butter.js" data-butter-exclude ></script>
+    <script type="text/javascript" src="external-media.js" data-butter-exclude ></script>
```
Comment (ScottDowne) [inline templates/external-media.html]: Same as above.

**Pull request merged**